citar.el

```diff
@@ -820,12 +819,27 @@ Return a list containing only (KEY . ENTRY) pairs."
                                           'invisible t))
       display-value)))
 
+(defun citar-format (template replacer)
+  "Format TEMPLATE with the function REPLACER.
+The templates are of form ${foo} for variable foo.
+REPLACER takes an argument of the format variable.
+Adapted from `citar-format'."
```

Comment:
Is this adapted from org-roam, or from s-format, @aikrahguzar?

Reply:
> Is this adapted from org-roam, or from s-format, @aikrahguzar?

`org-roam`. I didn't change the doc string apart from deleting the part about defaults.

Also this probably should be an internal function so `citar--format` maybe?

Reply:
> Also this probably should be an internal function so `citar--format` maybe?

Yes, that makes sense.